src/client/pfs_v2.go

```diff
@@ -58,11 +58,69 @@ func (c APIClient) NewFileOperationClientV2(repo, commit string) (_ *FileOperati
 	}); err != nil {
 		return nil, err
 	}
-	return &FileOperationClient{client: client}, nil
+	return &FileOperationClient{
+		client: client,
+		fileOperationCore: fileOperationCore{
+			client: client,
+		},
+	}, nil
+}
+
+// Close closes the FileOperationClient.
+func (foc *FileOperationClient) Close() error {
+	return foc.maybeError(func() error {
+		_, err := foc.client.CloseAndRecv()
+		return err
+	})
+}
+
+// CreateTmpFileSetClient is used to create a temporary fileset.
+type CreateTmpFileSetClient struct {
+	client pfs.API_CreateTmpFileSetClient
+	fileOperationCore
+}
+
+func (c APIClient) NewCreateTmpFileSetClient() (ret *CreateTmpFileSetClient, retErr error) {
+	defer func() {
+		retErr = grpcutil.ScrubGRPC(retErr)
+	}()
+	client, err := c.PfsAPIClient.CreateTmpFileSet(c.Ctx())
+	if err != nil {
+		return nil, err
+	}
+	return &CreateTmpFileSetClient{
+		client: client,
+		fileOperationCore: fileOperationCore{
+			client: client,
+		},
+	}, nil
+}
+
+// Close closes the FileOperationClient.
+func (foc *CreateTmpFileSetClient) CloseAndRecv() (*pfs.CreateTmpFileSetResponse, error) {
```

Comment:
I think `Close` is actually a better name for this method. The gRPC aspect should stay encapsulated in my opinion. Also, update the comment appropriately.

Reply:
Also `foc` is not really the appropriate variable name. The name for this abstraction might be a bit too wordy.